package.json

```diff
@@ -1,36 +1,37 @@
 {
-  "name": "react-basic-template",
-  "version": "1.0.0",
-  "description": "",
-  "main": "db.js",
-  "dependencies": {
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-2": "^6.24.1",
-    "express": "^4.16.4",
-    "react": "^16.8.3",
-    "react-dom": "^16.8.3",
-    "webpack": "^4.29.6",
-    "webpack-cli": "^3.2.3"
-  },
-  "babel": {
-    "presets": [
-      "react",
-      "stage-2"
-    ]
-  },
-  "devDependencies": {
-    "nodemon": "^1.18.11"
-  },
-  "scripts": {
-    "start": "node server/index.js",
-    "postinstall": "npm run webpack",
-    "webpack": "webpack",
-    "webpack:dev": "npm run webpack -- --mode=development --watch",
-    "start:dev": "npm run webpack:dev & DATABASE_URL=postgres://localhost/grace_shopper ./node_modules/.bin/nodemon server/index.js --ignore client/ --ignore dist/"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC"
+	"name": "grace-shopper-funky-monkey",
+	"version": "1.1.0",
+	"description": "",
+	"main": "db.js",
+	"dependencies": {
+		"babel-core": "^6.26.3",
```

Comment:
All `babel` and `webpack` dependencies are not `dependencies`, they are `devDependencies`.

Anything not used in your source code that clients or your server uses is not a dependency, it is a dependency for the developer to do work.

Reply:
Fixed.